GoogleForCreators/web-stories-wp
# Dashboard: Template Detail Redesign Updates

## Context

Updates the template details view to new figma redesign specs: https://www.figma.com/file/bMhG3KyrJF8vIAODgmbeqT/Design-System?node-id=467%3A13745 


## Summary

| Old      | New |
| ----------- | ----------- |
| ![Screen Shot 2021-03-03 at 10 08 06 AM](https://user-images.githubusercontent.com/10720454/109843516-61f59880-7c08-11eb-9d66-ce629d7b66a4.png) | ![Screen Shot 2021-03-03 at 10 07 46 AM](https://user-images.githubusercontent.com/10720454/109843543-691ca680-7c08-11eb-985e-3e0679532a36.png) |
| ![Screen Shot 2021-03-03 at 10 07 06 AM](https://user-images.githubusercontent.com/10720454/109843559-6e79f100-7c08-11eb-904d-5b26d36c45e8.png) | ![Screen Shot 2021-03-03 at 10 10 20 AM](https://user-images.githubusercontent.com/10720454/109843792-b3058c80-7c08-11eb-8cd5-a4d669f4aa95.png)
 |

## Relevant Technical Choices

- Nothing weird here, mostly deleting dead code and simplifying a little bit. (only 9 / 33 files changed here have any actual updates, everything else is just pruning dead code)
- Deleted bookmark placeholder stuff, that's super dead and when we bring it back it'll be easier to just do it fresh.
- Made sure heading outline is accurate and added some aria labeling with template title to be more useful. 
![Screen Shot 2021-03-03 at 9 51 54 AM](https://user-images.githubusercontent.com/10720454/109843911-db8d8680-7c08-11eb-9751-cd4fa7fdb777.png)


## To-do

- The dashboard grid PR I have up (#6588 ) has some consolidation of dashboard spacing values so when both of these are merged I can do a lot more clean up with static values here - like the padding and margin. 
- I'm not sure about the detail view spacing on smaller screens but that can be a follow up.  
- "Related Templates" isn't updated, it'll update with #6588 since it's using the shared `templateGrid`

## User-facing changes

- Design updates: Text elements, buttons, icons 

## Testing Instructions

- See that pagination still works as expected 
- See that 'use template' button still works as expected
- See that 'close' button still works as expected

### QA






This PR can be tested by following these steps:

1.

### UAT







This PR can be tested by following these steps:

1.

## Reviews

### Does this PR have a security-related impact?

No

### Does this PR change what data or activity we track or use?

No

### Does this PR have a legal-related impact?

No

## Checklist




---



Fixes #6596
